storagemarket/provider_loop.go

```diff
@@ -128,26 +138,31 @@ func (p *Provider) loop() {
 			p.wg.Add(1)
 			go func() {
 				defer p.wg.Done()
-				p.doDeal(deal, dealReq.dh)
+				p.doDeal(deal, dealReq.dh, false)
 				p.dealLogger.Infow(deal.DealUuid, "deal go-routine finished execution")
 			}()
 
 			dealReq.rsp <- acceptDealResp{&api.ProviderDealRejectionInfo{Accepted: true}, nil}
 
 		case publishedDeal := <-p.publishedDealChan:
 			deal := publishedDeal.deal
-			errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
+			pub, collat, errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
 			if errf != nil {
 				p.dealLogger.LogError(deal.DealUuid, "failed to untag funds", errf)
+			} else {
+				p.dealLogger.Infow(deal.DealUuid, "untagged funds for deal after publish", "untagged publish", pub, "untagged collateral", collat)
 			}
 			publishedDeal.done <- struct{}{}
 
 		case finishedDeal := <-p.finishedDealChan:
 			deal := finishedDeal.deal
 			p.dealLogger.Infow(deal.DealUuid, "deal finished")
-			errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
+			pub, collat, errf := p.fundManager.UntagFunds(p.ctx, deal.DealUuid)
 			if errf != nil && !xerrors.Is(errf, db.ErrNotFound) {
 				p.dealLogger.LogError(deal.DealUuid, "failed to untag funds", errf)
+			} else {
+				p.dealLogger.Infow(deal.DealUuid, "untagged funds for deal as deal finished", "untagged publish", pub, "untagged collateral", collat,
+					"err", errf)
```

Comment:
Same here: maybe we want to log something different / not log if errf is db.ErrNotFound